v2.3 REST API and postman collection changes

manuwelakanade-GSLAB opened the pull request:
v2.3 REST API and postman collection changes

manuwelakanade-GSLAB commented:
Hi @JosephPJordan : Can you please review and merge this pull request. Also edited conf.py to have 2.3 - please let me know if the version needs to be changed in any other file. Thank you.

JosephPJordan commented on docs/rest_api_ep.rst:
```diff
@@ -0,0 +1,274 @@
+REST API Dynamic Endpoint Attach Detach
+=======================================
+
+This section will go over the specifics of the REST APIs supported by F5 ACI ServiceCenter application for dynamic endpoint attach detach feature.
```
This section contains the specifics of the REST APIs supported by the F5 ACI ServiceCenter application  for the dynamic endpoint attach detach feature.

manuwelakanade-GSLAB replied:
Resolved in docs/rest_api.rst (docs/rest_api_ep.rst  is an outdated filename)

JosephPJordan commented on docs/rest_api_ep.rst:
```diff
@@ -0,0 +1,274 @@
+Request Headers
+---------------
+
+For all the F5 ACI ServiceCenter API calls following request header parameters are required:
```
For all the F5 ACI ServiceCenter API calls, the following request header parameters are required:

manuwelakanade-GSLAB replied:
Resolved in docs/rest_api.rst (docs/rest_api_ep.rst  is an outdated filename)

JosephPJordan commented on docs/rest_api_ep.rst:
```diff
@@ -0,0 +1,274 @@
+
+3. From the result of the POST, save the token returned
+
+4. In the “Headers” section of any further REST API requests, add a key-value pair. Key name is “DevCookie”, and its value should be the token obtained in the previous step
```
4. In the **Headers** section of any further REST API requests, add a key-value pair. Key name is **DevCookie**, and its value should be the token obtained in the previous step

manuwelakanade-GSLAB replied:
Resolved in docs/rest_api.rst (docs/rest_api_ep.rst  is an outdated filename)

JosephPJordan commented on docs/rest_api_2.rst:
```diff
@@ -0,0 +1,4156 @@
+REST API
+========
+
+This section will go over the specifies of the REST APIs supported by F5 ACI ServiceCenter application.
```
This section contains the specifics of the REST APIs supported by the F5 ACI ServiceCenter application.

manuwelakanade-GSLAB replied:
Resolved in docs/rest_api.rst (docs/rest_api_ep.rst  is an outdated filename)

JosephPJordan commented on docs/rest_api_2.rst:
```diff
@@ -0,0 +1,4156 @@
+Request Headers
+---------------
+
+For all the F5 ACI ServiceCenter API calls following request header parameters are required:
```
...API calls, the following request header...

manuwelakanade-GSLAB replied:
Resolved in docs/rest_api.rst (docs/rest_api_ep.rst  is an outdated filename)

JosephPJordan commented on docs/rest_api_2.rst:
```diff
@@ -0,0 +1,4156 @@
+
+3. From the result of the POST, save the token returned
+
+4. In the “Headers” section of any further REST API requests, add a key-value pair. Key name is “DevCookie”, and its value should be the token obtained in the previous step
```
replace quotes with ** ** to bold those words

manuwelakanade-GSLAB replied:
Resolved in docs/rest_api.rst (docs/rest_api_ep.rst  is an outdated filename)

JosephPJordan commented on docs/rest_api_2.rst:
```diff
@@ -0,0 +1,4156 @@
+Input Parameters
+----------------
+
+There are multiple F5 ACI ServiceCenter APIs which require input parameters to be retrieved for the APIC Logical Device. These input parameters can be retrieved from APIC object browser known as visore.html
```
...can be retrieved from the APIC object browser...

manuwelakanade-GSLAB replied:
Resolved in docs/rest_api.rst (docs/rest_api_ep.rst  is an outdated filename)

JosephPJordan commented on docs/rest_api_2.rst:
```diff
@@ -0,0 +1,4156 @@
++=============+================================================================================================================================+=========================================================================================================================================================+
+| ldev        | 1. Go to APIC GUI → Tenant→ Services→ L4-L7→ Devices→ <Your L4-L7 Device>                                                      |     uni/tn-Sample\_2/lDevVip-f5-gs                                                                                                                      |
+|             |                                                                                                                                |                                                                                                                                                         |
+|             | 2. Right click on the <Your L4-L7 Device> and click on “Open in Object Store Browser”                                          |                                                                                                                                                         |
```
remove both "on" and replace quotes with ** **, ie Right click the <your L4..., and click **Open in Object...

manuwelakanade-GSLAB replied:
Resolved in docs/rest_api.rst (docs/rest_api_ep.rst  is an outdated filename)

JosephPJordan commented on docs/rest_api_2.rst:
```diff
@@ -0,0 +1,4156 @@
+|             |                                                                                                                                |                                                                                                                                                         |
+|             | 2. Right click on the <Your L4-L7 Device> and click on “Open in Object Store Browser”                                          |                                                                                                                                                         |
+|             |                                                                                                                                |                                                                                                                                                         |
+|             | 3. Use the dn property from the object browser for input parameter “ldev”.                                                     |                                                                                                                                                         |
```
the input parameter

manuwelakanade-GSLAB replied:
Resolved in docs/rest_api.rst (docs/rest_api_ep.rst  is an outdated filename)

JosephPJordan commented on docs/rest_api_2.rst:
```diff
@@ -0,0 +1,4156 @@
++-------------+--------------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+
+| lif         | 1. Go to APIC GUI → Tenant→ Services→ L4-L7→ Devices→ <Your L4-L7 Device> → Cluster Interfaces→ <Your Interface of Interest>   |     uni/tn-Sample\_2/lDevVip-f5-gs/lIf-external                                                                                                         |
+|             |                                                                                                                                |                                                                                                                                                         |
+|             | 2. Right click on the logical interface and click on “Open in Object Store Browser”                                            |                                                                                                                                                         |
```
remove "on" in both places and replace "" with ** **

manuwelakanade-GSLAB replied:
Resolved in docs/rest_api.rst (docs/rest_api_ep.rst  is an outdated filename)

JosephPJordan commented on docs/rest_api_2.rst:
```diff
@@ -0,0 +1,4156 @@
+|             |                                                                                                                                |                                                                                                                                                         |
+|             | 1. Go to APIC GUI → Tenant→ Services→ L4-L7→ Devices→ <Your L4-L7 Device> → Cluster Interfaces→ <Your Interface of Interest>   |                                                                                                                                                         |
+|             |                                                                                                                                |                                                                                                                                                         |
+|             | 2. Right click on the logical interface and click on “Open in Object Store Browser”.                                           |                                                                                                                                                         |
```
remove "on"s and bold instead of quotes

manuwelakanade-GSLAB replied:
Resolved in docs/rest_api.rst (docs/rest_api_ep.rst  is an outdated filename)

JosephPJordan commented on docs/rest_api_2.rst:
```diff
@@ -0,0 +1,4156 @@
+|             |                                                                                                                                |                                                                                                                                                         |
+|             | 2. Right click on the logical interface and click on “Open in Object Store Browser”.                                           |                                                                                                                                                         |
+|             |                                                                                                                                |                                                                                                                                                         |
+|             | 3. Click on > arrow of the dn property to see it’s children                                                                    |                                                                                                                                                         |
```
Click the > arrow of the dn property to see its children.

manuwelakanade-GSLAB replied:
Resolved in docs/rest_api.rst (docs/rest_api_ep.rst  is an outdated filename)

manuwelakanade-GSLAB commented:
Hi @JosephPJordan : Thanks for the suggesting the changes. I have addressed all the comments in rest_api.rst file because that is the final name of the file. Please let me know if you have any further suggestions.
